src/features/user_profile/components/SettingsForm/SettingsForm.tsx

```diff
@@ -231,8 +232,9 @@ function SettingsFormGen({ userProfile, updateUserProfile }) {
                 )}
 
                 <Select
+                  data-testid="osmEditor"
                   alwaysShowPlaceholder
-                  value={localSettings.osmEditor}
+                  value={localSettings.osmEditor || DEFAULT_OSM_EDITOR}
```

Comment:
changes in the app logics should be in a separate branch merged to main and deployed on prod separately. If to leave as it is, tests will be broken until a new release